dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1125,6 +1173,77 @@
         }
       }
     },
+    "EntityComponentInformation": {
+      "description": "The object for entity component information.",
+      "type": "object",
+      "discriminator": "componentKind",
+      "properties": {
+        "componentKind": {
+          "description": "The component object kind.",
+          "type": "string",
+          "enum": [
+            "PrebuiltComponent",
+            "LearnedComponent",
+            "ListComponent"
+          ],
+          "x-ms-enum": {
+            "name": "componentKind",
```

Comment:
Would the name `EntityCompenentKind` here work?

Reply:
I think this would be redundant as the main object is called EntityComponetInformation anyway. 
I'm flexible here, but let's wait for the board feedback this week.

Reply:
Okay, fair. I updated it